frontend/src/components/MenuBar/MenuBar.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { StyleSheet } from "react-native";
+//import { TouchableOpacity, Text } from "react-native";
```

Comment:
There are a lot of comments and unused code, e.g. TouchableHighlight and Text are imported but not used. In general, unless there is some reason, I would get rid of unused code as when left, nobody knows why it is there and it tends to stay forever.